modules/tfm/zephyr/CMakeLists.txt

```diff
@@ -178,3 +171,54 @@ set_property(TARGET zephyr_property_target
 )
 
 zephyr_include_directories(${NRF_DIR}/include/tfm)
+
+# Default values from config_base.h in TF-M.
+set(PLATFORM_SERVICE_INPUT_BUFFER_SIZE     64)
+set(PLATFORM_SERVICE_OUTPUT_BUFFER_SIZE    64)
+set(PLATFORM_SP_STACK_SIZE                 0x500)
+set(PLATFORM_NV_COUNTER_MODULE_DISABLED    ${CONFIG_TFM_PROFILE_TYPE_MINIMAL})
+set(CRYPTO_ENGINE_BUF_SIZE                 ${CONFIG_TFM_CRYPTO_ENGINE_BUF_SIZE})
+set(CRYPTO_CONC_OPER_NUM                   ${CONFIG_TFM_CRYPTO_CONC_OPER_NUM})
+set(CRYPTO_RNG_MODULE_ENABLED              ${CONFIG_TFM_CRYPTO_RNG_MODULE_ENABLED})
+set(CRYPTO_KEY_MODULE_ENABLED              ${CONFIG_TFM_CRYPTO_KEY_MODULE_ENABLED})
+set(CRYPTO_AEAD_MODULE_ENABLED             ${CONFIG_TFM_CRYPTO_AEAD_MODULE_ENABLED})
+set(CRYPTO_MAC_MODULE_ENABLED              ${CONFIG_TFM_CRYPTO_MAC_MODULE_ENABLED})
+set(CRYPTO_HASH_MODULE_ENABLED             ${CONFIG_TFM_CRYPTO_HASH_MODULE_ENABLED})
+set(CRYPTO_CIPHER_MODULE_ENABLED           ${CONFIG_TFM_CRYPTO_CIPHER_MODULE_ENABLED})
+set(CRYPTO_ASYM_SIGN_MODULE_ENABLED        ${CONFIG_TFM_CRYPTO_ASYM_SIGN_MODULE_ENABLED})
+set(CRYPTO_ASYM_ENCRYPT_MODULE_ENABLED     ${CONFIG_TFM_CRYPTO_ASYM_ENCRYPT_MODULE_ENABLED})
+set(CRYPTO_KEY_DERIVATION_MODULE_ENABLED   ${CONFIG_TFM_CRYPTO_KEY_DERIVATION_MODULE_ENABLED})
+set(CRYPTO_IOVEC_BUFFER_SIZE               ${CONFIG_TFM_CRYPTO_IOVEC_BUFFER_SIZE})
+set(CRYPTO_NV_SEED                         ${CONFIG_TFM_CRYPTO_NV_SEED})
+set(CRYPTO_SINGLE_PART_FUNCS_DISABLED      0)
+set(CRYPTO_STACK_SIZE                      ${CONFIG_TFM_CRYPTO_PARTITION_STACK_SIZE})
+set(TFM_FWU_BUF_SIZE                       PSA_FWU_MAX_WRITE_SIZE)
```

Comment:
Why isn't this wrapped in ${}. Is this some kind of special variable type?

Reply:
just an unspecial kind of bug.